_i18n/es.yml

```diff
@@ -37,22 +61,24 @@ contact_page:
       usajobs: USAJOBS
       veterans_affairs: Veterans Affairs
     errors:
-      captcha_required: You must complete the spam prevention challenge. # TODO: Translate
+      captcha_required: Debe completar el desafío de prevención de spam. # TODO: Translate
     form_helpers:
-      select: "- Select -" # TODO: Translate
+      select_topic: "- Seleccione un tema -"
+      select_issue: "- Seleccione un problema específico -"
+      select_agency: "- Seleccione una solicitud de agencia -"
     labels:
       agency_application: Aplicación de agencia
       agency_application_all: Todas las aplicaciones
       agency_application_common: Aplicaciones comunes
       agency_application_other: Otras aplicaciones
-      description: Provide more detail about your question # TODO: Translate
+      description: Proporcione más detalles sobre su pregunta # TODO: Translate
       email: Correo electrónico
       first_name: Nombre
       last_name: Apellido
       pii_warning: "**No** incluya información sensible o de identificación en esta sección, como su nombre completo, fecha de nacimiento o número de seguro social."
-      subtopic: Issue # TODO: Translate
-      topic: Topic # TODO: Translate
-    reset: Clear form # TODO: Translate
+      subtopic: Problema
+      topic: Tema
+    reset: Forma clara # TODO: Translate
```

Comment:
remove the TODO here?

```suggestion
    reset: Forma clara
```

Reply:
I'd left these as TODO only because they weren't included in the translations that we received from the professional translator. These are translated using Google Translate and I'd expect only to be temporary.

Reply:
Updated texts and removed TODO in 729b3fe.